src/Dropdown.vue

```diff
@@ -26,21 +28,18 @@
   </div>
 </template>
 <script>
-import {coerce} from './utils/utils.js'
+import {toBoolean} from './utils/utils.js'
```

Comment:
@Chng-Zhi-Xuan Can you explain the rationale for 2f4f069?

Reply:
@acjh 

The `coerce` keyword has been depreciated in Vue 2.0. Thus it is logical to abstract out functions within the `coerce` object to avoid `import coerce`. 

Though, it may seem like it is done halfway as I did not abstract all the functions nor did I remove the existing `coerce` const. 
1. The commit was done as other team members require the updated util to be consistent with migration efforts (without having to resolve merge conflicts).
2. Existing coerce was kept to prevent any compilation errors that may occur outside team member's component migration branch.
3.  The other 2 functions within coerce are only used in `input.vue`, which may not be migrated over as it was not used. Thus did not abstracted it out within this commit.

Reply:
Thanks for the explanation.